src/main/java/org/springframework/data/couchbase/config/AbstractCouchbaseConfiguration.java

```diff
@@ -431,15 +431,11 @@ public CustomConversions customConversions() {
 	 * and {@link #couchbaseMappingContext(CustomConversions)}.
 	 *
 	 * @param cryptoManager
+	 * @param objectMapper
 	 * @return must not be {@literal null}.
 	 */
 	public CustomConversions customConversions(CryptoManager cryptoManager, ObjectMapper objectMapper) {
-		List<Object> newConverters = new ArrayList();
-		// The following
-		newConverters.add(new OtherConverters.EnumToObject(getObjectMapper()));
-		newConverters.add(new IntegerToEnumConverterFactory(getObjectMapper()));
-		newConverters.add(new StringToEnumConverterFactory(getObjectMapper()));
-		newConverters.add(new BooleanToEnumConverterFactory(getObjectMapper()));
+		List<Object> newConverters = additionalConverters();
```

Comment:
Just leave the newConverters as-is, then instead of calling the new additionalConverters() here, call

registerAdditionalConverters(List<Object> converters)

But rename registerAdditionalConverters to additionalConverters().

Reply:
Sure

Reply:
Changes are Done @mikereiche